### PR TITLE
#1103 Metric Selection for Recommender Evaluation

### PR DESCRIPTION
**Pull request by Sibgha360**
**What's in the PR**
* The Evaluation Metric to be used to plot learning curve can be changed from the drop down on the side panel. 

**How to test manually**
* Set up a recommender for your project
* Annotate
* Open the recommender tab
* it should now be possible to change the displayed metric via a dropdown above the y-axis

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [x] PR updates documentation

### CODE REVIEW

File: inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/chart/resources/BootSideMenu.js [hunk 1]
```diff
@@ -0,0 +1,453 @@
+/**
```
[the remark this answers is not in the document]
Comment (Sibgha360) [reply inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/chart/resources/BootSideMenu.js]: After discussion with Ute, I have changed the BootSideMenu panel to my own simpler drop down panel.

File: inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/chart/resources/BootSideMenu.css [hunk 2]
```diff
@@ -0,0 +1,251 @@
+/*
```
[the remark this answers is not in the document]
Comment (Sibgha360) [reply inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/chart/resources/BootSideMenu.css]: I copied but made some changes in the js and css files both.

**Comment by reckart** [comment]
@UWinch this one seems to be ready for review

**Comment by reckart** [comment]
@Sibgha360 the description does not explain how to manually test. New features should include documentation updates.

**Pull request merged**